src/containers/Passphrase.jsx

```diff
@@ -21,14 +21,24 @@ const mapStateToProps = state => ({
   passphrase: state.passphrase
 })
 
+const showSuccessThenReload = t => {
+  Alerter.info(t('PassphraseView.reload'))
+
+  setTimeout(() => {
+    // the token changes after a password change, so we need to reload
+    // the page to get the new one
+    window.location.reload()
+  }, 4000) // 4s, a bit longer than the alert message
```

Comment:
Setting the duration explicitly in Alerter.info would allow us to remove the comment.

```
const ALERT_DURATION = 3500
   Alerter.info(t('PassphraseView.reload'), { duration: ALERT_DURATION })

setTimeout(() => {...}, ALERT_DURATION + 500) 

```

Reply:
See https://github.com/cozy/cozy-settings/pull/269/commits/bf320b8bdb79bb421577a521f02e426730ede871